src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -113,6 +115,14 @@ class PipelineRunSchema(NamedSchema, table=True):
         ondelete="SET NULL",
         nullable=True,
     )
+    configured_model_version_id: UUID = build_foreign_key_field(
```

Comment:
Just to keep this in line with all our other schemas/models, this can simply be `model_version_id` right? Same for the relationship below and also the `StepRunSchema`

Reply:
Renamed